src/Core/Framework/DataAbstractionLayer/Cache/EntityCacheKeyGenerator.php

```diff
@@ -29,7 +29,7 @@ public static function buildStreamTag(string $id): string
     /**
      * @param string[] $areas
      */
-    public function getSalesChannelContextHash(SalesChannelContext $context, array $areas = []): string
+    public static function getSalesChannelContextHash(SalesChannelContext $context, array $areas = []): string
```

Comment:
this is a break for everyone extending/decorating this service. 

we should introduce new static methods and deprecate the existing ones for 6.8